lib/src/core/mod.rs

```diff
@@ -22,6 +22,8 @@ macro_rules! default_reference_data {
     };
 }
 
+// The evidence(.json) must align with the "attestation" definition in:
+// https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestation
```

Comment:
I dont think this change should be part of this PR.

Reply:
Sure, I will add this in a separate PR.